src/MobileRedirection.php

```diff
@@ -520,7 +522,7 @@ public function add_mobile_version_switcher_link() {
 			<script data-ampdevmode>
 			(function( { containerId, isCustomizePreview, notApplicableMessage } ) {
 				addEventListener( 'DOMContentLoaded', () => {
-					if ( isCustomizePreview || window.ampPairedBrowsingClient ) {
+					if ( isCustomizePreview || [ 'paired-browsing-non-amp', 'paired-browsing-amp' ].includes( window.name ) ) {
```

Comment:
Shouldn't this be IE compatible?

Reply:
Humm. That's a good question. It _could_ but it doesn't _have_ to be. Since this is only shown when `amp_is_dev_mode()` this means it's only when the user is logged-in. The WordPress admin no longer supports IE, so I don't think we need to worry about this. 

We should, however, make sure that JS that is served to normal frontend visitors is compatible with IE. And this is the case because `mobile-redirection.js` gets processed by Webpack/Babel. This JS does not, but that's OK I think.

Reply:
Er, I stand (self-)corrected. [Browser support](https://make.wordpress.org/core/handbook/best-practices/browser-support/) for WordPress `includes`:

> Internet Explorer >= 11

Nevertheless, see [Guidelines for ancient browser support: IE](https://make.wordpress.org/core/2020/03/27/guidelines-for-ancient-browser-support-ie/). 

So I think it is OK if this `script` fails to work properly in IE11.